### ISSUE TITLE
How did you generate depth maps and precomputed camera coordinate files?

### ISSUE THREAD

**Issue by chudur-budur** [vislearn/dsacstar#6]
The instruction says to download generated depth maps and precomputed camera coordinate files.

Did you generated those from the 7scenes sequence images?

If yes, how did you compute/generated?

Is there any code for that?

**Comment by ebrach** [comment]
There is no code available for generating the depth maps, we used a custom renderer for that. We extracted a scene mesh from the TSDF volumes that come with the 7Scenes dataset (using ParaView) and rendered these using the ground truth poses. 

Alternatively, we can also use the original depth maps of 7Scenes and register them to the RGB images, see eg. https://projet.liris.cnrs.fr/voir/activities-dataset/kinect-calibration.html

Generating camera coordinates from the depth maps is easy. Refer to the generation of scene coordinates in dataset.py and omit the transformation with the camera pose.

Best,
Eric
